internal/configprovider/parser_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Should we have on top as well?
```
// Copyright Splunk Inc.
```

Reply:
Ah good point, I was doing this initially on the OTel repo. Let me update that.